shared_libraries/orca_shared/recovery/shared_recovery.py

```diff
@@ -271,5 +271,5 @@ def post_entry_to_standard_queue(
 
 
 def get_request_method(request_method: RequestMethod):
-    if request_method != "new_job":
+    if request_method != "new_job" or request_method != "update_file":
```

Comment:
We should not change the original python file (shared_recovery.py in this case) when doing unit tests. All changes should be done in the test file.

Reply:
Undid change